gamification/process/update.py

```diff
@@ -135,6 +136,34 @@ def get_participant_objects():
     return participants
 
 
+def get_participant_objects_meta_review():
+    meta_review_participant = meta_review_participants.objects.all()
+    return meta_review_participant
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = meta_review_participants.objects.get(
+                        pos_in=meta_review_participants.pos_in)
+    pos_out_meta_review = meta_review_participants.objects.get(
+                        pos_out=meta_review_participants.pos_out)
+    neg_in_meta_review = meta_review_participants.objects.get(
+                        neg_in=meta_review_participants.neg_in)
+    neg_out_meta_review = meta_review_participants.objects.get(
+                        neg_out=meta_review_participants.neg_out)
+    if pos_in_meta_review > 0 or pos_out_meta_review > 0 or \
+       neg_in_meta_review > 0 or neg_out_meta_review > 0:
+        participant.meta_review_completed = True
```

Comment:
Is this field defined anywhere? Moreover, IMO this field should belong to gamification participant, not meta-review participant. Seems you are passing a meta-review participant to this function `update_participants_data_with_meta_review`